RegexpHeader/Example2/config.xml

```diff
@@ -17,7 +17,7 @@
     </module>
 
     <module name="RegexpHeader">
-        <property name="headerFile" value="config/java.header"/>
+        <property name="headerFile" value="java.header"/>
```

Comment:
Please send PR to checkstyle project

Reply:
Please send update to main repository 
`<property name="headerFile" value="${config.folder}/java.header"/>`

Extractor should not do any changes.
Config should stay unchanged.

GitHub workflow should define system variable `config.folder` and propagate it to checkstyle execution.